packages/nvidia_nat_core/src/nat/utils/io/yaml_tools.py

```
@@ -25,6 +25,99 @@
+    Args:
+        file_path: Path to the file to load.
+
+    Returns:
+        The file content as a string.
+
+    Raises:
+        FileNotFoundError: If the file does not exist.
+    """
+    path = Path(file_path)
+    if not path.exists():
+        raise FileNotFoundError(f"Prompt file not found: {file_path}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)